pkg/gardenlet/controller/backupbucket/backup_bucket_reconciler.go

```diff
@@ -109,6 +109,14 @@ func (r *reconciler) reconcileBackupBucket(backupBucket *gardencorev1beta1.Backu
 		return reconcile.Result{}, err
 	}
 
+	// TODO: This code can be removed in a future version.
+	if controllerutils.HasFinalizer(secret, gardencorev1beta1.ExternalGardenerNameDeprecated) {
+		if err := controllerutils.RemoveFinalizer(r.ctx, r.client, secret.DeepCopy(), gardencorev1beta1.ExternalGardenerNameDeprecated); err != nil {
+			backupBucketLogger.Errorf("Could not remove deprecated finalizer from Secret referenced in SecretBinding: %s", err.Error())
```

Comment:
```suggestion
			backupBucketLogger.Errorf("Could not remove deprecated finalizer from Secret referenced in BackupBucket: %s", err.Error())
```
Copy paste, I guess ;)

Reply:
Nice catch! Thank you.